cafebabel/templates/base.html

```diff
@@ -1,8 +1,28 @@
+{% from "_meta-macro.html" import meta without context %}
 <!doctype html>
 <meta charset=utf-8>
 <meta name=viewport content="width=device-width, minimum-scale=1, initial-scale=1, shrink-to-fit=no">
 <title>{% block meta_title %}pan-European news, features and opinions{% endblock %} – Cafebabel</title>
-<meta name=description content="{% block meta_description %}{% endblock %}">
+
+{% block metas %}
+  {{ meta('name=description', 'pan-European news, features and opinions') }}
```

Comment:
@pointbar last thing I would change is to only initiate blocks in `base` and put all metas from here to `home.html` template

Reply:
fixed